src/sc/sidechainrpc.cpp

```diff
@@ -13,6 +13,8 @@
 #include <main.h>
 #include <init.h>
 
+#define MAX_LOOP 100
```

Comment:
Please don't use `#define` (MACRO style) to define constants as this pollutes the preprocessor space and basically becomes a literal copied everywhere.
Use `static const` or better `constexpr`

Reply:
I agree with your suggestion.
Modified accordingly.